fabric.properties

```diff
@@ -0,0 +1 @@
+apiKey=0123456789012345678901234567890123456789
```

Comment:
```
$ ./gradlew clean check
…
ERROR - Crashlytics Developer Tools error.
java.lang.IllegalArgumentException: Crashlytics found an invalid API key: null.
…
```

This file has to be in the catroid folder.
Since you've to update it anyway, please add a newline to the end of the file. This will remove the 'No newline at end of file' info.